Framework/PythonInterface/plugins/algorithms/SNSPowderReduction.py

```diff
@@ -1480,12 +1308,16 @@ def _process_vanadium_runs(self, van_run_number_list, samRunIndex, **dummy_focus
             self.log().notice('Processing vanadium {}'.format(van_run_ws_name))
 
             # create the donor workspace for calculating the sample correction
-            absWksp = self._create_absorption_input(van_run_number,
-                                                    material={'ChemicalFormula': 'V', 'SampleNumberDensity': 0.0721},
-                                                    geometry={'Shape': 'Cylinder',
-                                                              'Height': 7.,
-                                                              'Radius': self._vanRadius,
-                                                              'Center': [0., 0., 0.]})
+            absWksp = AbsorptionCorrUtils.create_absorption_input(van_run_number, self._info,
+                                                                  self._num_wl_bins,
+                                                                  material={'ChemicalFormula': 'V',
+                                                                            'SampleNumberDensity': 0.0721},
```

Comment:
This is a good idea. I'm not sure if there is an existing place for constants in Python, so I just defined it in the utils file. It can be moved if there is a better place to put it

Reply:
Looks good to me